scripts/helper.js

```diff
@@ -26,7 +27,7 @@ exports.parseURL = parseURL;
 function parseURL(url) {
     var output = null;
     url = url || '';
-    if (url.indexOf(TR_URL) === 0 && url != "http://www.w3.org/TR/1999/WAI-WEBCONTENT-19990505") {
+    if (((url.indexOf(TR_URL) === 0) || (url.indexOf(TR_URL_HTTP) === 0)) && url != "http://www.w3.org/TR/1999/WAI-WEBCONTENT-19990505") {
```

Comment:
You probably need to have https://www.w3.org/TR/1999/WAI-WEBCONTENT-19990505 here too. :(